README.md

```diff
@@ -20,6 +20,15 @@ steps:
     directory: infra/tf
 ```
 
+Multiple directories can be set as input, separated by space:
```

Comment:
Super nits, but I feel like the flow would be better if we put this after `Specific Terraform version can be used:`, since that part still talks about a single directory case.